src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -69,14 +67,11 @@ internal class RecommendationService(
             surname = personDetails?.personalDetailsOverview?.surname,
             gender = personDetails?.personalDetailsOverview?.gender,
             ethnicity = personDetails?.personalDetailsOverview?.ethnicity,
-            dateOfBirth = personDetails?.personalDetailsOverview?.dateOfBirth,
-            mappa = riskResponse?.mappa,
-            addresses = personDetails?.addresses
```

Comment:
As discussed, think we should put this back in for the address

Reply:
Done.